test/integration/rendering-error-plan-failed/00-install.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: kudo.k8s.io/v1alpha1
+kind: Operator
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: invalid
+---
+apiVersion: kudo.k8s.io/v1alpha1
+kind: OperatorVersion
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: invalid-v1
+  namespace: default
+spec:
+  version: "1.0.0"
+  connectionString: ""
+  operator:
+    name: Toy
+    kind: Operator
+  templates:
+    serial.yaml: |
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        namespace: default
+        name: serial-{{ .StepName }}
+      spec:
+        template:
+          metadata:
+            name: "{{ .StepName }}-serial-{{ .StepNumber }}"
+          spec:
+            restartPolicy: OnFailure
+            containers:
+            - name: bb
+              image: busybox:latest
```

Comment:
These pods are (currently) being created against envtest so no actual containers are able to be created. If there's not a way to write this test without requiring containers to start, I'll setup a KIND environment to run our tests (either, a separate suite or just run everything in KIND).

Reply:
Oh, this shouldn't render at all, I see :)